### ISSUE TITLE
awscc_chatbot_slack_channel_configuration is removed in 0.7.0 version? 0.6.0 is fine

### ISSUE THREAD

**Issue by wreulicke** [hashicorp/terraform-provider-awscc#295]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the Cloudformation Open Coverage Roadmap.

### Terraform CLI and Terraform AWS Cloud Control Provider Version

```
$ terraform -v
Terraform v1.0.5
on linux_amd64
+ provider registry.terraform.io/hashicorp/aws v3.67.0
+ provider registry.terraform.io/hashicorp/awscc v0.7.0

Your version of Terraform is out of date! The latest version
is 1.0.11. You can update by downloading from https://www.terraform.io/downloads.html
```

### Affected Resource(s)

* awscc_chatbot_slack_channel_configuration

### Terraform Configuration Files

```hcl
resource "awscc_chatbot_slack_channel_configuration" "notification" {
  configuration_name = "deadletter-queue-notification"
  iam_role_arn       = aws_iam_role.chatbot.arn
  slack_channel_id   = var.slack_channel_id
  slack_workspace_id = var.slack_workspace_id

  sns_topic_arns = [
    var.sns_topic_arns
  ]
}
```

### Expected Behavior

We can define `awscc_chatbot_slack_channel_configuration`.

### Actual Behavior

We cannnot define `awscc_chatbot_slack_channel_configuration` by `Invalid resource type`.
We can define it If use awscc 0.6.0.

```
$ terraform plan
│ Error: Invalid resource type
│ 
│   on ../../modules/contents-optimizer/deadletter.tf line 53, in resource "awscc_chatbot_slack_channel_configuration" "notification":
│   53: resource "awscc_chatbot_slack_channel_configuration" "notification" {
│ 
│ The provider hashicorp/awscc does not support resource type "awscc_chatbot_slack_channel_configuration".
╵
```

### Steps to Reproduce

1. add `awscc_chatbot_slack_channel_configuration` resource 
2. `terraform apply`

### References

- Removed at this PR https://github.com/hashicorp/terraform-provider-awscc/pull/285

**Comment by wreulicke** [comment]
It seems this PR, https://github.com/hashicorp/terraform-provider-awscc/pull/285 removed the resource type.

**Comment by ewbankkit** [comment]
@wreulicke Thanks for raising this issue.
Yes, there must have been a problem downloading the CloudFormation resource type schema definition last time we generated the provider.
I can see that the definition is available and this resource will be included in the next release (probably today).

**Comment by ewbankkit** [comment]
`awscc_chatbot_slack_channel_configuration` is available with **v0.8.0** of the Terraform AWS Cloud Control Provider.
